modules/rosa-hcp-cluster-terraform-file-creation.adoc

```diff
@@ -97,11 +97,25 @@ module "rosa-hcp" {
   aws_subnet_ids         = var.create_vpc ? var.private_cluster ? module.vpc[0].private_subnets : concat(module.vpc[0].public_subnets, module.vpc[0].private_subnets) : var.aws_subnet_ids
   create_account_roles   = true
   create_operator_roles  = true
+# Optional: Configure a cluster administrator user <.>
```

Comment:
for hcp module
the version 1.6.2 should change to 1.6.3

Reply:
Thanks @marcolan018  - made that change on line 88 and also line 40 of this file.